src/AI/JakeMan.java

```diff
@@ -0,0 +1,770 @@
+package AI;
+
+import java.util.*;
+import java.util.Map.Entry;
+
+import AI.CommanderProductionInfo;
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderAbility;
+import Engine.*;
+import Engine.UnitActionLifecycles.WaitLifecycle;
+import Terrain.*;
+import Units.*;
+
+
+public class JakeMan extends ModularAI
+{
+  private static class instantiator implements AIMaker
+  {
+    @Override
+    public AIController create(Army army)
+    {
+      return new JakeMan(army);
+    }
+
+    @Override
+    public String getName()
+    {
+      return "JakeMan";
```

Comment:
Where does this name come from, anyway?

Reply:
Lost&Found frequently has similar things as his nickname in the AWBW server, and it's his decision tree.

Reply:
I mean, OK I guess. Might cause confusion with the similarly-named CO, but there's no actual affinity there. Not my baby though.